crates/ibc-apps/ics20-transfer/src/lib.rs

```diff
@@ -0,0 +1,27 @@
+//! Implementation of the IBC [fungible token transfer](https://github.com/cosmos/ibc/blob/main/spec/app/ics-020-fungible-token-transfer/README.md) (ICS-20) application logic.
+#![no_std]
+#![forbid(unsafe_code)]
+#![cfg_attr(not(test), deny(clippy::unwrap_used))]
+#![cfg_attr(not(test), deny(clippy::disallowed_methods, clippy::disallowed_types,))]
+#![deny(
+    warnings,
+    trivial_casts,
+    trivial_numeric_casts,
+    unused_import_braces,
+    unused_qualifications,
+    rust_2018_idioms
+)]
+#![allow(clippy::result_large_err)]
+
+#[cfg(any(test, feature = "std"))]
+extern crate std;
+
+#[doc(inline)]
+pub use ibc_app_transfer_types as types;
+
+#[cfg(feature = "serde")]
```

Comment:
In general, should all of the modules be configured with the `serde` feature? Asking so I know whether I need to add these to the `ibc-clients` crate.

Reply:
Yes actually, and in some cases even the `borsh`, `schema` and `parity-scale-codec` features.